RELEASE.rst

```diff
@@ -0,0 +1,9 @@
+RELEASE_TYPE: patch
```

Comment:
I think this should be a minor release – it’s expanding the public API, allowing sampling of something that wasn’t previously possible.

Reply:
👍